### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by bobbybalow1**
Finished assignment. Please review.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -33,89 +33,121 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
...
- action 3:
+ Noun: glass
+ action 1: glass holds water
+ action 2: glass holds beer
+ action 3: glass holds wine
  */
-
+glass.water();
+glass.beer();
+glass.wine();
```
Comment (matkatmusic) [inline main.cpp]: `water` is not an action
`holdWater` is the action.
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -33,89 +33,121 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
...
- action 2:
- action 3:
+ Noun: pan
+ action 1: pan cooks chicken
+ action 2: pan cooks beef
+ action 3: pan cooks veggies
  */
+pan.chicken();
+pan.beef();
+pan.veggies();
```
Comment (matkatmusic) [inline main.cpp]: these are not actions
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -33,89 +33,121 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
...
- action 3:
+ Noun: cutting board
+ action 1: cut chicken
+ action 2: cut carrots
+ action 3: cut celery 
  */
 
+cuttingBoard.chicken();
+cuttingBoard.carrots();
+cuttingBoard.celery();
```
Comment (matkatmusic) [inline main.cpp]: these are not actions.
- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -33,89 +33,121 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
...
- action 3:
+ Noun: asparagus
+ action 1: throw away
+ action 2: eat
+ action 3: kill with fire
  */
 
+asparagus.throwAway();
+asparagus.eat();
+asparagus.killWithFire();
```
Comment (matkatmusic) [inline main.cpp]: some other object performs these actions on or to asparagus.  Asparagus does not perform these actions
- [ ] revise

File: main.cpp [hunk 5]
```diff
@@ -33,89 +33,121 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
...
- action 3:
+ Noun: cpp Homework
+ action 1: Watch
+ action 2: Comeplete assignment
+ action 3: Push
  */
 
+cppHomework.watch();
+cppHomework.completeAssignment();
+cppHomework.push();
```
Comment (matkatmusic) [inline main.cpp]: some other object performs these actions on cppHomework.  cppHomework does not perform these actions.
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp [hunk 6]
```diff
@@ -61,20 +61,20 @@ pan.veggies();
...
+ action 2: check for new email
+ action 3: delete email
  */
 
-ipad.checkEmail();
-ipad.watchMovies();
-ipad.doWork();
+email.send();
+email.checkNew();
+email.trash();
```
Comment (matkatmusic) [inline main.cpp]: an email **client** does these things

**Review by matkatmusic — APPROVED** [review]

File: main.cpp (hunk 6 above)
Comment (bobbybalow1) [reply main.cpp]: Good point! Thanks Chuck.

**Pull request merged**